documentation/src/docs/security/pravega-security-configurations.md

```diff
@@ -9,46 +9,65 @@ You may obtain a copy of the License at
-|pravegaservice.secureZK|Enable TLS for communication between segment store and Apache Zookeeper| False|
-|bookkeeper.zkTrustStore| Truststore for TLS communication between segment store and Apache Zookeeper| Empty |
-
-## Pravega Controller <a name ="pravega-controller"></a>
-
-|Parameter|Details|Default Value|
-|---------|-------|-------------|
-|config.controller.server.authorizationEnabled|Enable authorization/authentication| False|
-|config.controller.server.tlsEnabled|Enable encrypted channel between Pravega client and controller|False|
-|config.controller.server.tlsKeyFile|The key file (Private key) for communication between Pravega client and controller|Empty|
-|config.controller.server.tlsCertFile|Public key certificate for communication between Pravega client and controller|Empty|
-|config.controller.server.tokenSigningKey|Signing key used to sign the delegation token passed on to the segment store|Empty|
-|config.controller.server.userPasswordFile|File containing user details for default _auth_ implementation for Pravega (similar to `/etc/passwd`)|Empty|
-|config.controller.server.zk.secureConnectionToZooKeeper|Enable TLS for connection to Apache ZooKeeper| False|
-|config.controller.server.zk.trustStorePath|Truststore for TLS communications with Apache ZooKeeper| False|
-
-## Pravega Standalone
-For ease of use Pravega standalone hides some of the configurations that are mentioned above. Below is the table containing relevant configurations for Pravega Standalone:
-
-|Parameter|Details|Default Value|
-|---------|-------|-------------|
-|singlenode.enableTls|Enable TLS between all the components deployed within the singlenode| False|
-|singlenode.enableAuth|Enable authentication/authorization between all the components within the singlenode |False|
-|singlenode.certFile|If TLS is enabled, the public key certificate is used for internal communication between segment store and controller|Empty|
-|singlenode.keyFile|If TLS is enabled, this represents the private key by all the server sockets| Empty|
-|singlenode.passwdFile|If _auth_ is enabled, this represents the password file for the default _auth_ plugin implementation|Empty|
-|singlenode.userName|If _auth_ is enabled, this represents the username used for internal communication between segment store and controller|Empty|
-|singlenode.passwd|If _auth_ is enabled, this represents the password used for internal communication between segment store and controller|Empty|
+This document describes the security configuration parameters for Pravega, in both distributed and standalone modes.
+
+## Security Configuration Parameters in Distributed Mode
+
+In the distributed mode, Controllers and Segment Stores are configured individually. The following sub-sections describe
+their Transport Layer Security (TLS) and Auth (Authentication and Authorization) parameters.
+
+
+### Segment store <a name = "pravega-segment-store"></a>
+
```

Comment:
<a name....> not required.......may i know wat's the purpose of it.?...i can help u

Reply:
Done.